tools/docker/paraviewweb/Dockerfile

```diff
@@ -1,4 +1,53 @@
 
+#
+# The images built from this Dockerfile are designed to be used in one of two
+# ways, either run directly with a fully external application (www, server, and
+# launcher directories), or else extended by another image (see, for example,
+# the "demo" or "visualizer" directories in this repo).
+#
+# In order to run this image directly as a paraviewweb application, there are
+# a couple assumptions we make about the external directories.  You should first
+# create a directory structure where your application lives (we'll call it
+# <APP_ROOT> below) following this structure:
+#
+#     <APP_ROOT>/
+#     <APP_ROOT>/www/
+#     <APP_ROOT>/launcher/config.json
+#
```

Comment:
We are missing the must-be values. (i.e.: proxy-map file path, possible log files inside docker, pvpython path inside the docker for the cmd part...)